test/lib/ufe/testChildFilter.py

```diff
@@ -116,6 +118,60 @@ def testFilteredChildren(self):
         self.assertEqual(6, len(children))
         self.assertIn(ball3Item, children)
 
+    def testFilteredClassPrims(self):
+        classPrimFileName = testUtils.getTestScene('classPrims', 'class-prims.usda')
+        shapeNode, stage = mayaUtils.createProxyFromFile(classPrimFileName)
+        self.assertTrue(stage)
+        cmds.select(clear=True)
+
+        # Check that we see the 2 active prims
+        topPathStr = shapeNode + ',/top'
+        topPath = ufe.PathString.path(topPathStr)
+        topItem = ufe.Hierarchy.createItem(topPath)
+        topHier = ufe.Hierarchy.hierarchy(topItem)
+        self.assertEqual(2, len(topHier.children()))
+
+        # Get the child filter for the hierarchy handler corresponding
+        # to the runtime of the top item.
+        usdHierarchyHandler = ufe.RunTimeMgr.instance().hierarchyHandler(topItem.runTimeId())
+        childrenFilters = usdHierarchyHandler.childFilter()
+
+        # The prims that can be found in the file.
+        activeConeItem = ufe.Hierarchy.createItem(ufe.PathString.path(topPathStr + '/active_cone'))
+        activeCubeItem = ufe.Hierarchy.createItem(ufe.PathString.path(topPathStr + '/active_cube'))
+        inactiveCylinderItem = ufe.Hierarchy.createItem(ufe.PathString.path(topPathStr + '/inactive_cylinder'))
+        activeClassItem = ufe.Hierarchy.createItem(ufe.PathString.path(topPathStr + '/active_class'))
+        inactiveClassItem = ufe.Hierarchy.createItem(ufe.PathString.path(topPathStr + '/inactive_class'))
+
+        # Declare different filter settings that will be tested.
+        FilterSettings = collections.namedtuple('FilterSettings', ['filters', 'expecteditems'])
+        filterSettings = [
+            FilterSettings(
+                filters={ 'InactivePrims': False, 'ClassPrims': False },
```

Comment:
This is my thinking, that a full list of possibilities may not be known in case there's a need to include all.
Perhaps we can document the supported filters somewhere to be used as a reference.